hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -687,7 +692,7 @@ private void completeWhenRetryRejected(RejectedExecutionException e) {
         if (context.logger.isFinestEnabled()) {
             context.logger.finest(e);
         }
-        complete(new HazelcastInstanceNotActiveException(e.getMessage()));
+        notifyError(new HazelcastInstanceNotActiveException(e.getMessage()));
```

Comment:
Similar to `IndeterminateOperationStateException ` case, we want to complete immediately, without any further check.

Reply:
:+1: